contracts/Vault.vy

```diff
@@ -1720,7 +1731,7 @@ def report(gain: uint256, loss: uint256, _debtPayment: uint256) -> uint256:
     # Update reporting time
     self.strategies[msg.sender].lastReport = block.timestamp
     self.lastReport = block.timestamp
-    self.lockedProfit = gain  - totalFees  # profit is locked and gradually released per block
+    self.lockedProfit = self._calculateLockedProfit() + gain - totalFees
```

Comment:
This is adding locked profits from previous reports?

Reply:
yes. It adds any locked profits that have not been previously released. To stop reverse sandwich attack from price jumping up when new harvest prematurely releases a lot of profit

Reply:
Nice, very smart